### PR TITLE
Update release.yml

### PR DESCRIPTION
**Pull request by hipstersmoothie**
- Updated to checkout@v2 - all we need to do is fetch tags and history
- removed unecassary autorc options
- update auto

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -39,4 +32,4 @@ jobs:
         run: |
           yarn install --frozen-lockfile
           yarn build
-          yarn release
+          yarn release -vv
```
Comment (hipstersmoothie) [inline .github/workflows/release.yml]: added verbose logging to hopefully diagnose the issue

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -7,17 +7,10 @@ jobs:
...
 
       - name: Prepare repository
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          git checkout ${GITHUB_REF:11} --
-          git remote rm origin
-          git remote add origin "https://x-access-token:$GH_TOKEN@github.com/ericclemmons/codelift.git"
-          git fetch origin --tags
-          git branch --set-upstream-to origin/${GITHUB_REF:11} ${GITHUB_REF:11}
```
Comment (hipstersmoothie) [inline .github/workflows/release.yml]: This git scripts was to do something that's fixed in v2

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -7,17 +7,10 @@ jobs:
...
+        run: git fetch --prune --unshallow
```
Comment (hipstersmoothie) [inline .github/workflows/release.yml]: v2 only checkout head/branch commit so we need to fetch the full history. This is from their docs

File: .autorc [hunk 4]
```diff
@@ -1,8 +1,4 @@
 {
-  "repo": "codelift",
-  "owner": "ericclemmons",
-  "name": "Eric Clemmons",
-  "email": "eric@smarterspam.com",
```
Comment (hipstersmoothie) [inline .autorc]: All of this stuff should come from your package.json. Including it here just makes it kinda confusing

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -34,9 +27,9 @@ jobs:
 
       - name: Create Release
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (hipstersmoothie) [inline .github/workflows/release.yml]: Support for GITHUB_TOKEN was added in https://github.com/intuit/auto/pull/1036. This should also fix the url stuff.

File: package.json [hunk 6]
```diff
@@ -19,10 +19,10 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@auto-it/all-contributors": "^9.17.1",
-    "@auto-it/first-time-contributor": "^9.17.1",
+    "@auto-it/all-contributors": "^9.19.1",
```
Comment (hipstersmoothie) [inline package.json]: `v9.19.1` contains the GitHub actions. fixes

File: .autorc (hunk 4 above)
Comment (ericclemmons) [reply .autorc]: You're absolutely right.  When I first started this my `package.json` was much more sparse!

File: .github/workflows/release.yml (hunk 3 above)
Comment (ericclemmons) [reply .github/workflows/release.yml]: Cross-reference for future:
> https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

File: .github/workflows/release.yml (hunk 5 above)
Comment (ericclemmons) [reply .github/workflows/release.yml]: You don't waste any time do ya ;)

> Screen Shot 2020-03-08 at 2 57 50 PM

**Comment by ericclemmons** [comment]
Thanks so much @hipstersmoothie ! Going to merge this in & monitor the next release 🤞

(Unrelated, I'm wondering how I would create a preview release for a contributor's PR, such as this one: https://github.community/t5/GitHub-Actions/Run-a-GitHub-action-on-pull-request-for-PR-opened-from-a-forked/m-p/31147#M690)

**Comment by hipstersmoothie** [comment]
Woot woot! 🎉

**Pull request merged**